### PR TITLE
Make changes on Trigger for good work

### PR DESCRIPTION
**Pull request by jgomezr**
Make change on 1 line on trigger for survey 4.

### CODE REVIEW

File: src/triggers/AlsurCreateFarmerProductDeliveryComplete.trigger [hunk 1]
```diff
@@ -4,6 +4,7 @@ trigger AlsurCreateFarmerProductDeliveryComplete on Farmer_Product_Delivery__c (
 		if (newItem.TypeOfDelivery__c == 'Productor - entrega IGUAL a asignación') {
 			for(Farmer_Order_Allocation__c foa : [SELECT Number_of_Baskets__c, Delivery_Date__c, Number_of_Kilos__c, Number_of_Units_of_Presentation__c, Presentacion__c FROM Farmer_Order_Allocation__c WHERE Id =:newItem.Farmer_Order_Allocation__c]){ 
 				Farmer_Product_Delivery__c delivery = new Farmer_Product_Delivery__c();
```
Comment (lc0rp) [inline src/triggers/AlsurCreateFarmerProductDeliveryComplete.trigger]: Isn't newItem already a Farmer_Product_Delivery object?

If so, you should just be adding the fields the newItem directly.

newItem.Number_of_Baskets__c = fao.Number_of_baskets__c;
newItem.Delivery_Date__c = fao.Delivery_Date__c;
etc ...

and then the trigger will take care of saving it. You don't even have to worry about saving anything...

Comment (lc0rp) [inline src/triggers/AlsurCreateFarmerProductDeliveryComplete.trigger]: You would need to change the trigger to "before insert"

**Comment by jgomezr** [comment]
action as we call it in update trigger, the trigger calls me or some unique ID field to update, so I made the switch, I went by the evidence that error and I solved it by just

**Comment by jgomezr** [comment]
Now make some changes for the trigger operate properly

**Pull request merged**